elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -156,6 +164,12 @@
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition AUTHENTICATION_CONTEXT_ATTRIBUTE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.AUTHENTICATION_CONTEXT, ModelType.STRING, true)
```

Comment:
Since `authentication-context` is a required attribute, the last parameter here should be `false` (since this attribute isn't optional).

Reply:
fixed